build-chef-tarball

```diff
@@ -68,6 +71,15 @@ echo "chef_version: $chef_version"
 echo "chef_dir: $chef_dir"
 echo "jobs: $jobs"
 echo "tarball: $tarball"
+echo "global_zone: ${GZ:-false}"
+
+# If we're building for the global zone, relocate GCC libraries to remove
+# /opt/local dependencies
+if [[ -n $GZ ]]; then
```

Comment:
Since `gz` should *only* be `true` or `false`, and its value is controlled 100% by this program at runtime, it's simpler to be written as:

``` bash
if $gz; then
  foo
fi
```

https://github.com/bahamas10/bash-style-guide#quoting